packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -132,7 +132,7 @@ export function createEditPageNode({
         let parentPath: string | null = null;
 
         if (parentNodeData?.pageTreeNode) {
-            parentPath = parentNodeData.pageTreeNode.slug === "home" ? "/home" : parentNodeData.pageTreeNode.path;
+            parentPath = parentNodeData.pageTreeNode.slug === "home" && parentNodeData.pageTreeNode.path ? "/home" : parentNodeData.pageTreeNode.path;
```

Comment:
Why do you need this check?

Reply:
if the slug is home the parent path will be set to home even if the correct path would be test/home. This is not needed for the functionality but i decided to added for correctness

Reply:
It will not be empty but for example when creating a sub page for a page called Test should produce the /test for the `parentPath`variable but if the new page is called Home and the slug is `home` the parentPath will become `/home`

Reply:
Thanks for the explanation! Could you please introduce a local variable `hasParentNode` to make this more obvious?